chemprop/data/datasets.py

```diff
@@ -114,24 +114,25 @@ def normalize_targets(self, scaler: StandardScaler | None = None) -> StandardSca
         return scaler
 
     def normalize_inputs(
-        self, key: str | None = "X_f", scaler: StandardScaler | None = None
+        self, key: str | None = "X_d", scaler: StandardScaler | None = None
     ) -> StandardScaler:
-        VALID_KEYS = {"X_f", None}
+        VALID_KEYS = {"X_d", None}
         if key not in VALID_KEYS:
             raise ValueError(f"Invalid feature key! got: {key}. expected one of: {VALID_KEYS}")
 
-        X = self.X_f
+        X = self.X_d
 
         if scaler is None:
-            scaler = StandardScaler().fit(X)
+            scaler = StandardScaler()
+            self.X_d = scaler.fit_transform(X)
 
         return scaler
 
     def reset(self):
         """Reset the {atom, bond, molecule} features and targets of each datapoint to its
         initial, unnormalized values."""
```

Comment:
```suggestion
        """Reset the atom and bond features; atom and extra descriptors; and targets of each
        datapoint to their initial, unnormalized values."""
```

Same thing for `MoleculeDataset().reset()` on line 285.